public/app/plugins/datasource/loki/components/monaco-query-field/monaco-completion-provider/completions.ts

```diff
@@ -98,6 +98,21 @@ const UNWRAP_FUNCTION_COMPLETIONS: Completion[] = [
   },
 ];
 
+const LOGFMT_ARGUMENT_COMPLETIONS: Completion[] = [
+  {
+    type: 'FUNCTION',
+    label: 'strict',
```

Comment:
have you considered using `--strict` as label. I personally think it would make it more understandable and it would also make autocomplete better
<img width="1099" alt="image" src="https://github.com/grafana/grafana/assets/30407135/d73966cf-2b27-4e64-92f4-b70eba5e1c69">

<img width="1417" alt="image" src="https://github.com/grafana/grafana/assets/30407135/ff6f5642-2d3f-4287-80d3-06634c3d90be">

Reply:
Interesting. I would expect the autocomplete to pick up `-`, but it's probably generating another new path for logfmt. I'll iterate a bit on this 👍

Reply:
Done 
![Uploading imagen.png…]()